en/canary-deployment-tidb-operator.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Canary Deployment of TiDB Operator
+summary: Learn how to canary upgrade TiDB Operator and how to deploy multiple TiDB Operator to manage different TiDB clusters.
+---
+
+# Canary Deployment of TiDB Operator
+
+This document describes how to canary upgrade TiDB Operator and how to deploy multiple TiDB Operator to manage different TiDB clusters.
+
+## Scenarios
+
+- Canary upgrade TiDB Operator.
+- Deploy multiple TiDB Operator to manage different TiDB clusters.
+
+    > **Note:**
+    >
+    > - Currently, you can only deploy multiple `tidb-controller-manager`s and `tidb-scheduler`s.
+    > - Only one Admission Webhook is required.
+    > - If you have deployed multiple TiDB Operator and only some of them enable [Advanced StatefulSet](advanced-statefulset.md), the same TidbCluster Custom Resource (CR) cannot be switched among these TiDB Operator.
+
+## Related parameters
+
+To support canary deployment, the following parameters are added to the `values.yaml` file in the `tidb-operator` chart:
+
+- `appendReleaseSuffix`
+
+    If this parameter is set to `true`, when you deploy TiDB Operator, TiDB Operator adds automatically a prefix (`-{{ .Release.Name }}`) to the name of resources related to `tidb-controller-manager` and `tidb-scheduler`.
+
+    For example, if you execute `helm install canary pingcap/tidb-operator ...`, the name of the `tidb-controller-manager` deployment is `tidb-controller-manager-canary`.
+
+    If you need to deploy multiple TiDB Operator, set this parameter to `true`.
+
+    Default value: `false`.
+
+- `controllerManager.create`
+
+    Controls whether to create `tidb-controller-manager`.
+
+    Default value: `true`.
+
+- `controllerManager.selector`
+
+    Sets the `-selector` parameter of `tidb-controller-manager`. The parameter is used to filter the CRs controlled by `tidb-controller-manager` according to the CR labels. If multiple selectors exist, the selectors are in `and` relationship.
+
+    Default value: `[]` (`tidb-controller-manager` controls all CRs).
+
+    Example:
+
+    ```yaml
+    selector:
+    - canary-release=v1
+    - k1==v1
+    - k2!=v2
+    ```
+
+- `scheduler.create`
+
+    Controls whether to create `tidb-scheduler`.
+
+    Default value: `true`.
+
+## Deploy multiple TiDB Operator to manage different TiDB clusters
+
+1. Deploy the first TiDB Operator.
+
+    Refer to [Deploy TiDB Operator](deploy-tidb-operator.md) to deploy the first TiDB Operator. Add the following configuration in the `values.yaml`:
+
+    ```yaml
+    controllerManager:
+      selector:
+      - user=dev
+    ```
+
+2. Deploy the TiDB cluster.
+
+    1. Refer to [Configure the TiDB Cluster](configure-a-tidb-cluster.md) to configure the TidbCluster CR, and configure `labels` to match the `selector` set in the last step. For example:
```

Comment:
labels or label?

Reply:
In the yaml below it's `labels`, so I took the liberty to change it to `labels`.
@DanielZhangQD PTAL.

Reply:
`labels` is OK.